app/visualisation_methods/pareto/plugin.rb

```diff
@@ -82,12 +82,14 @@ def get_pareto_data(moes)
       sum_min =0
       sum_max =0
       simulation_runs.map do |datas|
-        if datas[:arguments][arg_name] ==local_max
-          sum_max+=datas[:result]
-        end
+        unless datas[:result].blank?
+          if datas[:arguments][arg_name] ==local_max
```

Comment:
Jak zwykle czepianie się, ale była dużo razy mowa o konwencji formatowania. Przed i po operatorze powinna być spacja, inaczej to się zlewa w jeden ciąg znaków. Jeżeli jest tylko po jednej stronie, to wygląda nawet gorzej. Mówię o tym tutaj i wszystkim poniżej.